static/js/tutorial.js

```diff
@@ -388,7 +388,7 @@ function finale() {
         }
 
         if (stream_data.in_home_view("social")) {
-            send_delayed_stream_message("social", "cute animals", "This is a message on stream `social` with the topic `cute animals`. Try uploading or pasting in some pictures. Here's a [guinea pig](https://humbug-user-uploads.s3.amazonaws.com/byqgM1qjol1mzje_KzeNRT5F/guinea.jpg) to get you started:", 75);
+            send_delayed_stream_message("social", "cute animals", "This is a message on stream `social` with the topic `cute animals`. Try uploading or pasting in some pictures. Here's a [guinea pig](/static/images/cute/guinea.jpg) to get you started:", 75);
```

Comment:
You mean relative path, right?  Well I tested it and it worked, and I think it makes sense for it to; and it's simpler than concatenating settings.EXTERNAL_HOST onto it...